### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS3.2.1][IntelliJ][Wipro]: Combo box is closed automatically while navigating to the list items in "Container Registry" combo box.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1449]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app for containers.
4. "Run on web app for Containers" dialog will open.
5. Tab to "Container registry" combo box the expand it using 'Alt+down arrow' then select list items and observe the combo box is closed

**Expected Result:**
Using down arrow, list item should be selected and should not close  the dropdown without any user interference.

**Actual Result:**
After expanding the combo box  ,when we traverse to the next element drop down is getting closed and focus is moving to the "Image and tag" edit box.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![after selecting the combo box focus in webapp containers](https://user-images.githubusercontent.com/22956130/35730256-371f6046-07c6-11e8-8bc2-30168f941fdb.png)

**Comment by Eskibear** [comment]
Will be shipped in v3.7.0 . To verify the issue now, download the latest dev build [HERE](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/lastSuccessfulBuild/Azure/)

**Comment by WiproPSAccTest** [comment]
We are not able to install the private build in our machine.
Please find the screenshot 
![intellij-private build](https://user-images.githubusercontent.com/22956130/39748642-39dd16b8-52ce-11e8-8fdf-8bc3a2b5263a.jpg)

**Comment by Eskibear** [comment]
@WiproPSAccTest 
There're 3 links in the [page](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/lastSuccessfulBuild/Azure/) , from top down, namely for `Eclipse`, `IJ 2017`, `IJ 2018`. You should click the middle one if you are using IntelliJ 2017.

Please check if you have downloaded the correct one.

**Comment by WiproPSAccTest** [comment]
Verified in V.3.7.0 .Issue got fixed.
Please add closed tag
![1449-fixed](https://user-images.githubusercontent.com/22956130/39859034-1743c1a4-5456-11e8-85b0-ca394de1d8bd.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749890
